Configuration_sample_local.hpp

```diff
@@ -43,8 +43,12 @@
 
 // TMC2209 UART settings
 // These settings work only with TMC2209 in UART connection (single wire to TX)
-#define RA_RMSCURRENT 1200       // RMS current in mA. Warning: Peak current will be 1.414 times higher!! Do not exceed your steppers max current!
-#define DEC_RMSCURRENT 1000      // RMS current in mA. Warning: Peak current will be 1.414 times higher!! Do not exceed your steppers max current!
+#define RA_MOTOR_CURRENT_RATING 0       // Current rating of RA motor in mA
```

Comment:
Why is this zero? If we are expecting users to set this, we should require it in the config_local and #ifndef error here

Reply:
The user is required to edit this according to their specific motor rating. It's required when using UART, so it should be defined as 'something' by default - zero works. ConfigurationValidation will throw an error if it remains at zero or is >1700 (the maximum output for a TMC2209).

I've added an #ifndef error to ConfigurationValidation in case someone removes it from the local config.